templates/btax/header.html

```diff
@@ -0,0 +1,32 @@
+{% load staticfiles %}
```

Comment:
yes.  this all started from a big cut and paste.  I'll reference existing file instead.